src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,23 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(Folder *folder, const SyncFileItemPtr &item)
```

Comment:
@FlexW Please add **const** before **Folder** so we demonstrate a clear intent of not modifying an instance the ***folder** is pointing to.